core/src/tag_utils.rs

```diff
@@ -52,6 +56,7 @@ impl SwfMovie {
     pub fn from_movie_and_subdata(&self, data: Vec<u8>, source: &SwfMovie) -> Self {
         Self {
             header: self.header.clone(),
+            total_bytes: self.total_bytes - self.data.len() + data.len(),
```

Comment:
This function is only used when reading clip events as a sorta-hack, because `swf` currently returns clip events as a Vec instead of a slice from the overall SWF input.  So this should actually return the standard `self.total_bytes`, because,e.g. doing `getBytesTotal()` in a `onClipEvent(enterFrame)` should return the length of the overall movie.